cdap-api/src/main/java/co/cask/cdap/api/app/ApplicationConfigurer.java

```diff
@@ -35,72 +31,23 @@
 /**
  * Configures a CDAP Application.
  */
-public interface ApplicationConfigurer {
+public interface ApplicationConfigurer extends DatasetConfigurer {
+
   /**
-   * Sets the Application's name.
+   * Sets the name of the Application.
    *
-   * @param name The Application name
+   * @param name name
    */
   void setName(String name);
 
   /**
-   * Sets the Application's description.
+   * Sets the description of the Application.
    *
-   * @param description The Application description
+   * @param description description
    */
   void setDescription(String description);
 
   /**
-   * Adds a {@link Stream} to the Application.
-   *
-   * @param stream The {@link Stream} to include in the Application
-   */
```

Comment:
We should not remove them. Please don't remove APIs this way. We have to deprecate and follow the standard process.